lib/ransack/adapters.rb

```diff
@@ -0,0 +1,42 @@
+module Ransack
+  module Adapters
+
+    def self.current_adapters
+      @current_adapters ||= {
+        :active_record => defined?(::ActiveRecord::Base),
+        :mongoid => defined?(::Mongoid) && !defined?(::ActiveRecord::Base)
+      }
+    end
+    def self.require_constants
+      require 'ransack/adapters/mongoid/ransack/constants' if current_adapters[:mongoid]
+      require 'ransack/adapters/active_record/ransack/constants' if current_adapters[:active_record]
```

Comment:
Line is too long. [100/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.